.github/workflows/publish.yml

```diff
@@ -0,0 +1,36 @@
+# Based on
+# https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/
+
+name: Publish Python distributions to PyPI
+
+on: push
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python distributions to PyPI
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.9
+    - name: Install pypa/build
+      run: >-
+        python -m
+        pip install
+        build
+        --user
+    - name: Build a binary wheel and a source tarball
+      run: >-
+        python -m
+        build
+        --sdist
+        --wheel
+        --outdir dist/
+        .
+    - name: Publish distribution to PyPI
+      if: startsWith(github.ref, 'refs/tags')
```

Comment:
Can't we move this check to the push trigger to prevent this workflow to run at all when not pushing to a tag?

Reply:
I was going to suggest this, but it might be good to run it until here occasionally, even if we aren't making a release. That way we will notice if the release build breaks.

Reply:
Yes, this serves as a test that building the package still works.